locale/fr/foreman.po

```diff
@@ -7356,8 +7356,8 @@ msgstr "État d’alimentation"
 msgid "Power a Virtual Machine"
 msgstr "Démarrer une Machine Virtuelle"
 
-msgid "Power has been set to \"%s\" successfully"
-msgstr "L’alimentation a pu être définie sur \"%s\""
+msgid "Power has been set to \\\"%s\\\" successfully"
```

Comment:
@ekohl I made this change. Is this what you were saying to do?

Reply:
> @ekohl I made this change. Is this what you were saying to do?

This is indeed what I meant.

> `pl/foreman.po:11990: 'msgid' and 'msgstr' entries do not both end with '\n'`

That's a warning from a different file. I left a comment there (that I updated it).